docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -1053,7 +1058,7 @@ yb-voyager import schema --export-dir <EXPORT_DIR> \
         --post-snapshot-import true
 ```
 
-If any of the CREATE INDEX DDLs fail in the preceding command, drop the INVALID indexes on the target database using:
+If any of the CREATE INDEX DDLs fail in the preceding command, retry the command with the argument `--ignore-exist` to ignore already created indexes and create new ones instead as follows:
```

Comment:
I believe we can delete the sql block below and just have the content : 
```
If any of the CREATE INDEX DDLs fail in the preceding command, retry the command with the argument --ignore-exist to ignore already created indexes and create new ones instead
```
cc: @sanyamsinghal  pls confirm.